packages/manager/modules/pci/src/components/project/storages/databases/database.class.js

```diff
@@ -89,6 +90,16 @@ export default class Database extends Base {
     return this.currentEngine;
   }
 
+  getSSLModeKey() {
```

Comment:
This method might be called multiple times by the digest loop. Because of this the method includes will iterate the array multiple times. Is it possible to do this check once at instance level or only when something is changed? if not its ok with me. Its might improve negligible performance.

Reply:
It's a good point. Fixed :)